compiler/rustc_middle/src/ty/normalize_erasing_regions.rs

```diff
@@ -7,9 +7,12 @@
 //! `normalize_generic_arg_after_erasing_regions` query for each type
 //! or constant found within. (This underlying query is what is cached.)
 
-use crate::ty::fold::{TypeFoldable, TypeFolder};
 use crate::ty::subst::{Subst, SubstsRef};
 use crate::ty::{self, Ty, TyCtxt};
+use crate::{
+    mir,
+    ty::fold::{TypeFoldable, TypeFolder},
+};
```

Comment:
```suggestion
use crate::mir;
use crate::ty::fold::{TypeFoldable, TypeFolder};
use crate::ty::subst::{Subst, SubstsRef};
use crate::ty::{self, Ty, TyCtxt};
```

Reply:
oh... heh... maybe I can teach rust-analyzer to stop doing that

Reply:
You can use `"rust-analyzer.assist.importMergeBehavior": "last"`.